lib/windows.js

```diff
@@ -146,6 +200,7 @@ function windows() {
 	return {
 		platform: 'windows',
 		title: windowTitle,
+		processPath,
```

Comment:
?

Reply:
Whoops, didn't mean to push that up, cleaning up.